dev.exs

```diff
@@ -58,6 +58,8 @@ defmodule SamplePhoenixWeb.Router do
 
   pipeline :api do
     plug :accepts, ["json"]
+    plug Accent.Plug.Request, default_case: Accent.Case.Snake
+    plug Accent.Plug.Response, default_case: Accent.Case.Camel, json_codec: Jason
```

Comment:
If we go down this route we'll need to:
- create a `BeaconWeb.API.Plug` as https://hexdocs.pm/plug/Plug.Builder.html
- update https://github.com/BeaconCMS/beacon/blob/main/guides/introduction/api.md to instruct users to add `plug BeaconWeb.API.Plug` in the `pipeline :api` block in theirs apps' router.
- and then replace these 2 lines with `plug BeaconWeb.API.Plug` as well

Reply:
Ready. It's way neater!

Reply:
Yeah and the main reason for that change is to avoid exposing 3rd-party code that we have no control over.